sentry-rails/lib/sentry/rails/action_cable/connection.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require_relative 'exception_reporter'
+
+module Sentry
+  module Rails
+    module ActionCable
+      module Connection
+        private
+
+        def handle_open
```

Comment:
I thought this has been defined in `sentry-rails/lib/sentry/rails/action_cable.rb`?